packages/it/index.js

```diff
@@ -35,24 +35,24 @@ function standardiseDescriptor(descr) {
   var deepConf = descr.deepConf;
 
   var p = isObject(props) || isObject(properties) ?
-    assign({}, props, properties) : undefined;
+    assign(assign({}, props), properties) : {};
```

Comment:
I remembered now. This (and similar) change is against the specification. I have to reject this PR unfortunately. The root cause is different.

Reply:
Just to clarify, the above line attempt to addreeses two issues:

1. the `assign()` function currently accepts two parameters, thus (unless `assign()` is to be refactored) in order to process three or more parameters, calls to `assign()` need to be nest by pair of parameters.
1. the last parameters to be `undefined` or `{}` for which I should probably investigate more.

Do you agree about the first point and reject the second?

Reply:
@koresar I believe I have worked out the correct fix now.

```
  var sdp = isObject(deepStatics) ? merge({}, deepStatics) : undefined;
  sdp = isObject(staticDeepProperties) ? merge(sdp || {}, staticDeepProperties) : sdp;
```

Currently `merge()` throws when the first argument is undefined, hence the `sdp || {}`

Also wondering if `merge({}, deepStatics)` couldn't be simplified with `Object.assign(deepStatics)`